.claude/skills/markdown-formatter/SKILL.md

```
@@ -0,0 +1,183 @@
+
+### Step 1: 自動整形スクリプトの実行（初期修正）
+
+プロジェクトに用意されている自動フォーマットスクリプトを実行し、基本的な見出し前後の空行や末尾改行などを自動的に一括修正します。
+
+```bash
+bun scripts/format-markdown.mjs <file_path>
+```
+
+> [!NOTE]
+> `scripts/format-markdown.mjs` は MD022, MD012, MD047 のルールを自動で修正しますが、リストの内部にあるネストされたコードブロック等の MD031 (blanks-around-fences) については自動修正の対象外です。これらは手動で修正を行い、`git diff` で変更点を確認する必要があります。
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)